int32 gripper_angle

